.github/workflows/markdown-lint-fix.yml

```
@@ -0,0 +1,33 @@
+name: Daily Markdown lint fix PR
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+
+      - name: Lint markdown files
+        run: |
+          npm i -g markdownlint-cli
+          markdownlint "**/*.md" --fix
```

We don't want it to spend time on `node_modules`.
```suggestion
          markdownlint "./files/**/*.md" "./meta/**/*.md" --fix
```

---

There is no npm/yarn install before this is run, so there is no need. The better why though, if you're running locally is to use `-i node_modules` to ignore the folder